charts/opentelemetry-demo/values.yaml

```diff
@@ -641,16 +644,31 @@ opentelemetry-collector:
         enable_open_metrics: true
 
     processors:
-      # Make use of the spanmetrics processor to compute RED metrics from all tracing spans
+      # we can remove when this is fixed: https://github.com/open-telemetry/opentelemetry-demo/issues/737
+      transform:
+        metric_statements:
+          - context: metric
+            statements:
+              - set(description, "Measures the duration of inbound HTTP requests") where name == "http.server.duration"
+      # we can remove when this is fixed: https://github.com/open-telemetry/opentelemetry-demo/issues/826
+      filter:
+        metrics:
+          exclude:
+            match_type: strict
+            metric_names:
+              - queueSize
```

Comment:
Let's use the filterprocessor config that uses OTTL since the other config will be deprecated eventually.

Reply:
It'll be something like

```
processors:
  filter/ottl:
    error_mode: ignore
    metrics:
      metric:
          - 'name == "queueSize"'
```